README.md

```diff
@@ -32,10 +32,12 @@ NextAuth.js is a complete open source authentication solution for [Next.js](http
 
 It is designed from the ground up to support Next.js and Serverless.
 
-This is the core repo for NextAuth.js. Check the repos below if you are interested in additional information:
+This is a monorepo containing the following packages / projects:
 
-- Docs related: https://github.com/nextauthjs/docs
-- Adapter related: https://github.com/nextauthjs/adapters
+1. The primary `next-auth` package
+2. A development test application
+3. All `@next-auth/adapter-*` packages
```

Comment:
```suggestion
3. All `@next-auth/*-adapter` packages
```

We actually suffix them as npm packages, it's jut their folder name that uses a prefix :D